Client/Frontend/Browser/Onboarding/OnboardingRewardsAgreementView.swift

```diff
@@ -0,0 +1,231 @@
+                    .font: UIFont.systemFont(ofSize: 16, weight: UIFont.Weight.regular),
+                    .foregroundColor: UX.linkColor,
+                    .link: "brave_terms_of_service"
+                ]))
+                
+                text.append(NSAttributedString(string: ".", attributes: [
+                    .font: UIFont.systemFont(ofSize: 16, weight: UIFont.Weight.regular),
+                    .foregroundColor: titleLabel.textColor!
+                ]))
+                
+                let paragraphStyle = NSMutableParagraphStyle()
+                paragraphStyle.lineBreakMode = .byWordWrapping
+                
+                text.addAttribute(.paragraphStyle, value: paragraphStyle, range: NSRange(location: 0, length: text.length))
+                
+                return text
+            }()
+            
+            descriptionLabel.accessibilityLabel = descriptionLabel.text
+            descriptionLabel.accessibilityTraits = [.staticText, .link]
+            descriptionLabel.accessibilityValue = nil
+            descriptionLabel.isAccessibilityElement = true
+        }
+        
+        init(theme: Theme) {
+            super.init(frame: .zero)
+            
+            applyTheme(theme)
+            mainStackView.tag = OnboardingViewAnimationID.details.rawValue
+            descriptionStackView.tag = OnboardingViewAnimationID.detailsContent.rawValue
+            imageView.tag = OnboardingViewAnimationID.background.rawValue
+            
+            addSubview(imageView)
+            addSubview(mainStackView)
+            mainStackView.snp.makeConstraints {
+                $0.leading.equalTo(self.safeArea.leading)
+                $0.trailing.equalTo(self.safeArea.trailing)
+                $0.bottom.equalTo(self.safeArea.bottom)
+            }
+            
+            descriptionView.addSubview(descriptionStackView)
+            descriptionStackView.snp.makeConstraints {
+                $0.edges.equalToSuperview().inset(UX.descriptionContentInset)
+            }
+            
+            mainStackView.addArrangedSubview(descriptionView)
+
+            [skipButton, agreeButton, UIView.spacer(.horizontal, amount: 0)]
+                .forEach(buttonsStackView.addArrangedSubview(_:))
+            
+            [textStackView, buttonsStackView].forEach(descriptionStackView.addArrangedSubview(_:))
+            
+            descriptionCheckbox.addTarget(self, action: #selector(onTermsAccepted(_:)), for: .touchUpInside)
+        }
+        
+        func applyTheme(_ theme: Theme) {
+            descriptionView.backgroundColor = OnboardingViewController.colourForTheme(theme)
+            titleLabel.textColor = theme.isDark ? .white : .black
+            updateDescriptionLabel()
+            if theme.isDark {
+                descriptionCheckbox.setImage(#imageLiteral(resourceName: "checkbox_off_dark"), for: .normal)
```

Comment:
can be simplified to `let image = theme.isDark ? img1 : img2`
and then set the image